index.js

```diff
@@ -613,6 +613,10 @@ TChannelConnection.prototype.handleReqFrame = function handleReqFrame(reqFrame)
 
     var handler = self.localEndpoints[name] || self.channel.endpoints[name];
 
+    self.channel.emit('endpoint', {
```

Comment:
Let's please not emit for:
- invalid handler
- local handler

In fact you should probably emit misses specially.

Let's for now do something like:
- inside the typeof not function branch, emit something (maybe same with diff args?)
- add an `else if (handler !== self.localEndpoints[name]) { /* emit */` branch; in v2 this becomes simply an `else`